pyprotista/parsers/ident/comet_2020_01_4_parser.py

```diff
@@ -110,22 +171,9 @@ def check_parser_compatibility(cls, file):
         contains_engine = "Comet" in head
         return is_mzid and contains_engine
 
-    def _map_mods_and_sequences(self):
```

Comment:
cannot comment on line 105 but no need for file.as_posix()

Reply:
The check on whether or not that file is a comet file should 
a) be done on the headers only (no need to iter 10 lines)
b) if iter of the file use for line in f - no need to call next
c) no need to reset the head variable in python as it exists only in scope and is recreated every time

I would suggest to use 
```
 with open(file) as i:
      header = i.readline()
if "Comet:EValue" in header
```
to be perfectly explicit

Reply:
i dont think the header is in the first line (xml yada yada), and the try except is in place because you would wanna avoid crashes in empty files (e.g. a 0 PSM MS Amanda file)

Reply:
as posix is still in there :)
`with open(file.as_posix()) as f:` will crash on windows - why add as_posix to path lib object? Also Doc string say file is str so str.as_posix won't work at all :)